ember-app/app/views/milestones.js

```diff
@@ -8,6 +8,7 @@ var MilestonesView = Ember.View.extend({
     this.$(".board").sortable({
       handle: 'h3',
       placeholder: "milestone-placeholder",
+      axis: "x",
```

Comment:
this caused really weird issues when I was testing it. It caused the milestone to carry under and disappear